.github/workflows/ci-macos.yml

```diff
@@ -24,9 +24,9 @@ jobs:
           brew install chezscheme
           brew install coreutils
           echo "::add-path::$HOME/.idris2/bin"
-      - name: Build and test Idris 2 from bootstrap
+      - name: Build Idris 2 from bootstrap
         run: make bootstrap && make install
         shell: bash
-      - name: Build and test self-hosted
-        run: make clean && make all && make test INTERACTIVE=''
+      - name: Test self-hosted
```

Comment:
Self-hosted means compiled using itself as the compiler. By removing this step
you are testing the executable compiled using the bootstrap, not using itself.

Reply:
On the makefile it was already doing that.

stage2 does a clean and rebuild.

Reply:
Ah wait, NVM.

So:

Bootstrap -> HEAD -> HEAD so it is important to check that HEAD can build HEAD.

Reply:
Ok @gallais, amended and added back as a separate stage to be able to compare time difference on self-host compile and tests.